kernel/main.c

```diff
@@ -340,6 +353,116 @@ async_init(void)
       panic("Unable to create a kthread for do_async_init()");
 }
 
+static void
+setup_uefi_runtime_services(void)
+{
+   struct mem_region ma;
+
+   if (!booting_with_uefi)
+      return;
+
+   EFI_RUNTIME_SERVICES *RT = TO_PTR(uefi_rt_addr);
+   printk("Runtime Services: %p\n", RT);
+
+   int num_entries = 0;
+
+   /*
+    * When SetVirtualAddressMap() is called, the physical
+    * addresses should be directly accessible. So, identity
+    * map them.
+    */
+   for (int i = 0; i < get_mem_regions_count(); i++) {
+      get_mem_region(i, &ma);
+
+      if (ma.type != TILCK_BOOT_EFI_RUNTIME_CODE && ma.type != TILCK_BOOT_EFI_RUNTIME_DATA)
```

Comment:
Line too long, probably way over 80 cols. You can transform it this way:
```
if (ma.type != TILCK_BOOT_EFI_RUNTIME_CODE &&
    ma.type != TILCK_BOOT_EFI_RUNTIME_DATA)
{
   continue;
}
```

Note the braces and the fact that the opening brace is on a new-line, contrary to all the other cases. That's because the expression has been split: https://github.com/vvaltchev/tilck/blob/master/docs/contributing.md